src/Controller/EventController.php

```diff
@@ -83,8 +83,14 @@ public function invite(Event $event, User $user, EntityManagerInterface $entityM
      */
     public function show(Event $event): Response
     {
+        $participants = $event->getParticipants();
+        $isParticipant = false;
+        if (in_array($this->getUser(), $participants->getValues())) {
+            $isParticipant = true;
+        }
```

Comment:
try this:

    $isParticipant = in_array($this->getUser(), $participants->getValues());

Reply:
Done ! Smooth, should have thought about that :)